### PR TITLE
Update configure OpenShift OAuth

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Update configuration OpenShift OAuth

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18152

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

screenshot-0 0 0 0_4000-2021 02 15-11_50_35

### CODE REVIEW

File: modules/administration-guide/partials/proc_configuring-openshift-oauth.adoc [hunk 1]
```diff
@@ -10,18 +10,36 @@ For users to interact with OpenShift, they must first authenticate to the OpenSh
...
 
 .Procedure
 
-* OpenShift OAuth will be enabled by default, deploy {prod-short} using xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[OperatorHub] or the {prod-cli}, see the
-link:{link-cli-github}#user-content-{prod-cli}-serverdeploy[`{prod-cli} server:deploy` specification] chapter.
+* Configure Openshift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
++
+====
+If skip this step and there are identity providers configured on the OpenShift cluster, then {prod} will create initial OpenShift user for HTPasswd identity provider and put user's credentials in `openshift-oauth-user-credentials` secret in {prod-namespace} namespace. Follow these steps to obtain credentials to log in into OpenShift cluster and {prod}:
```
Comment (AndrienkoAleksandr) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth.adoc:25]:
```suggestion
If skip this step and there no are identity providers configured on the OpenShift cluster, then {prod} will create initial OpenShift user for HTPasswd identity provider and put user's credentials in `openshift-oauth-user-credentials` secret in {prod-namespace} namespace. Follow these steps to obtain credentials to log in into OpenShift cluster and {prod}:
```

Comment (AndrienkoAleksandr) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth.adoc]:
```suggestion
If skip this step and there are no identity providers configured on the OpenShift cluster, then {prod} will create initial OpenShift user for HTPasswd identity provider and put user's credentials in `openshift-oauth-user-credentials` secret in {prod-namespace} namespace. Follow these steps to obtain credentials to log in into OpenShift cluster and {prod}:
```

File: modules/administration-guide/partials/proc_configuring-openshift-oauth.adoc [hunk 2]
```diff
@@ -10,18 +10,36 @@ For users to interact with OpenShift, they must first authenticate to the OpenSh
...
 
 * The `oc` tool is available.
 * `{prod-cli}` management tool is available. See xref:overview:using-the-chectl-management-tool.adoc[].
-* Openshift identity providers are configured on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration]
 
 .Procedure
 
-* OpenShift OAuth will be enabled by default, deploy {prod-short} using xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[OperatorHub] or the {prod-cli}, see the
-link:{link-cli-github}#user-content-{prod-cli}-serverdeploy[`{prod-cli} server:deploy` specification] chapter.
+* Configure Openshift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth.adoc]:
```suggestion
* Configure OpenShift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
```

File: modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* Configure Openshift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
++
+====
+If skip this step and there are identity providers configured on the OpenShift cluster, then {prod} will create initial OpenShift user for HTPasswd identity provider and put user's credentials in `openshift-oauth-user-credentials` secret in {prod-namespace} namespace. Follow these steps to obtain credentials to log in into OpenShift cluster and {prod}:
+
+. Obtain OpenShift user name:
++
+[subs="+attributes,+quotes"]
+----
+oc get secret openshift-oauth-user-credentials  -n {prod-namespace} -o json | jq -r '.data.user' | base64 -d
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc]:
```suggestion
$ oc get secret openshift-oauth-user-credentials  -n {prod-namespace} -o json | jq -r '.data.user' | base64 -d
```

File: modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+----
+. Obtain OpenShift user password:
++
+[subs="+attributes,+quotes"]
+----
+oc get secret openshift-oauth-user-credentials -n {prod-namespace} -o json | jq -r '.data.password' | base64 -d
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc]:
```suggestion
$ oc get secret openshift-oauth-user-credentials -n {prod-namespace} -o json | jq -r '.data.password' | base64 -d
```

File: modules/administration-guide/partials/proc_removing-initial-openshift-user.adoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
+[id="proc_removing-initial-openshift-user_{context}"]
+== Removing OpenShift initial OAuth user
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_removing-initial-openshift-user.adoc]:
```suggestion
= Removing OpenShift initial OAuth user
```

File: modules/administration-guide/partials/proc_configuring-openshift-oauth-without-initial-user.adoc [hunk 6]
```diff
@@ -0,0 +1,38 @@
+[id="proc_configuring_openshift-oauth-without-initial-user_{context}"]
+== Configuring OpenShift OAuth without provisioning OpenShift initial OAuth user
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth-without-initial-user.adoc]:
```suggestion
= Configuring OpenShift OAuth without provisioning OpenShift initial OAuth user
```

Comment (yhontyk) [reply modules/administration-guide/partials/proc_configuring-openshift-oauth-without-initial-user.adoc]: Suggest having 1st level headings here so the headings match:
Screenshot from 2021-02-16 09-44-15

File: modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+= Configuring OpenShift OAuth
+
+.Prerequisites
+
+* The `oc` tool is available.
+* `{prod-cli}` management tool is available. See xref:overview:using-the-chectl-management-tool.adoc[].
+
+.Procedure
+
+* Configure Openshift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
```
Comment (yhontyk) [inline modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc]:
```suggestion
* Configure OpenShift identity providers on the cluster. See the link:https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider[Understanding identity provider configuration].
```

**Review by yhontyk — APPROVED** [review]

**Review by MichalMaler — APPROVED** [review]

**Pull request merged**